### PR TITLE
feat(components): more prominent relative growth advantage value display

### LINKED ISSUE #231: Relative Growth Advantage: Improve display of advantage metrics
In CoV Spectrum, the growth advantage value is displayed like this:

grafik

Make a suggestion to make the growth advantage value more visible. Currently, the value is displayed quite small below the diagram:
grafik
It's the most important value here, it should be easy to see.

### PR DESCRIPTION
**Pull request by JonasKellerer**
resolves #231

### Summary

Increases the visibility and prominence of the relative growth advantage value.

### Screenshot

grafik

grafik

### PR Checklist

~~- [ ] All necessary documentation has been adapted.~~
- [x] The implemented feature is covered by an appropriate test.

### CODE REVIEW

**Review by chaoran-chen — APPROVED** [review]
Thanks! I noticed two things:

- There are two percentage signs ("68.96%%")
- The confidence interval should be put in parentheses.

File: components/src/preact/relativeGrowthAdvantage/relative-growth-advantage-chart.tsx [hunk 1]
```diff
@@ -52,14 +53,37 @@ const RelativeGrowthAdvantageChart = ({ data, yAxisScaleType }: RelativeGrowthAd
...
+    relativeAdvantageLowerBound: number;
+    relativeAdvantageUpperBound: number;
+}) => {
+    return (
+        <div class='mx-auto flex items-end flex-wrap'>
+            <span class='text-[#606060]'>Relative advantage:</span>
+            <div>
+                <span class='text-2xl ml-3'> {formatProportion(relativeAdvantage)}% </span>
+                <span class='ml-2.5'>
+                    {formatProportion(relativeAdvantageLowerBound)}% - {formatProportion(relativeAdvantageUpperBound)}%
```
Comment (chaoran-chen) [inline components/src/preact/relativeGrowthAdvantage/relative-growth-advantage-chart.tsx:84]: There are now two percentage signs ("68.96%%")

Comment (chaoran-chen) [inline components/src/preact/relativeGrowthAdvantage/relative-growth-advantage-chart.tsx:84]: The confidence interval should be put in parentheses.

Comment (JonasKellerer) [reply components/src/preact/relativeGrowthAdvantage/relative-growth-advantage-chart.tsx]: :see_no_evil: True.

**Pull request merged**